results/final_results.json

```diff
@@ -64,6 +64,11 @@
         "reward_low_FP_rate": 27.08121005969828,
         "standard": 35.687043255862065
     },
+    "spatial": {
+        "reward_low_FN_rate": 52.56449422487971,
```

Comment:
please add these results in the Readme tables, along with a similar comment as above.